storage/local/storage.go

```diff
@@ -212,10 +212,18 @@ func (s *memorySeriesStorage) Start() error {
 		return err
 	}
 	s.persistence = p
+	go s.persistence.run()
+
+	pclose := func() {
```

Comment:
I don't feel strong about it, so I'll change it.

What always irks me, is that you cannot overwrite an existing `err` with `err := f()` but with `a, err := g()`.